conf.py

```diff
@@ -39,7 +39,7 @@
 master_doc = sconf.master_doc
 language = 'en'
 project = sconf.project
-copyright = u'2008-{0}'.format(datetime.date.today().year)
+copyright = u'MongoDB, Inc. 2008-present. MongoDB, Mongo, and the leaf logo are registered trademarks of MongoDB, Inc'
```

Comment:
You just need the string to be 2008-present. 
We add all the MongoDB blurb in the jinja template when we build the html automatically.

Reply:
Just a little background -- because python docs don't use the jinja template, their conf.py is a little different, whereas since php-library uses our jinja template, just the date range is all we need.

Reply:
Fixed!

Reply:
Works for me!  👍